shared/tsconfig.base.json

```diff
@@ -34,9 +36,9 @@
     "noUnusedParameters": true,
     "outDir": "./.tsOuts/.tsOut/emit",
     "resolveJsonModule": true,
-    "sourceMap": true,
+    "sourceMap": false,
```

Comment:
This change looks scary - what does it do? will it make debugging harder?

Reply:
We don't use tsc for code gen